Added version control delete branch and hard reset checkpoints documentation (#299)
playcanvas/developer.playcanvas.com: content/en/user-manual/version-control/branches.md

![Close branch][4]

> However, if you have completed work on a branch you can close the branch which will remove it from the interface.

Author: ```suggestion
If you have completed work on a branch you can close the branch which will remove it from the the open branches list.
```

> If you have completed work on a branch you can close the branch which will remove it from the the open branches list.

Reviewer: ```suggestion
If you have completed work on a branch, you can close it which will remove it from the open branches list.
```

> To close a branch open the version control panel, select the branch you wish to close and choose the "Close this branch" option in the branch's drop-down menu. Note, you cannot close your current branch or the master branch. Switch to a different branch first if you wish to close your current branch.

Reviewer: ```suggestion
To close a branch, open the version control panel, select the branch you wish to close and choose the "Close this branch" option in the branch's drop-down menu. Note that you cannot close your current branch or the master branch. Switch to a different branch first if you wish to close your current branch.
```

![Close branch dialog][5]

You will be asked to confirm the closing of the branch and you have an option to discard any changes that were made to your branch since you last took a checkpoint. By default, PlayCanvas will save your changes in an additional checkpoint before closing the branch. If you wish to discard these changes you can select the option here.

**Note, enabling this checkbox will lose any work you have made in the branch since you last made a checkpoint**.

## Deleting a branch

Deletion of branches are only supported if the following conditions are met:

> - The branch as not been merged into another branch

Reviewer: ```suggestion
- The branch has not been merged into another branch
```
